src/fidesops/ops/models/privacy_request.py

```diff
@@ -138,6 +138,14 @@ def generate_request_callback_jwe(webhook: PolicyPreWebhook) -> str:
     return generate_jwe(json.dumps(jwe.dict()), config.security.app_encryption_key)
 
 
+def get_days_left(self: PrivacyRequest) -> Union[int, None]:
+    if self.due_date is None:
+        return None
+
+    delta = self.due_date.date() - datetime.now(timezone.utc).date()
+    return delta.days
```

Comment:
I would rename `self` here. `self` makes it look like it is part of the class but it isn't so I initially thought it wasn't indented correctly.

You could also slightly simplify the utc date with `utcnow`.

```suggestion
def get_days_left(privacy_request: PrivacyRequest) -> Union[int, None]:
    if privacy_request.due_date is None:
        return None

    delta = privacy_request.due_date.date() - datetime.utcnow().date()
    return delta.days
```